.travis/osx..install.sh

```diff
@@ -2,18 +2,18 @@
 
 PACKAGES="cmake pkgconfig fftw libogg libvorbis libsndfile libsamplerate jack sdl stk portaudio node fltk"
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
 	PACKAGES="$PACKAGES homebrew/versions/qt55"
 else
 	PACKAGES="$PACKAGES cartr/qt4/qt"
 fi
 
-brew install $PACKAGES ccache
+brew install "$PACKAGES" ccache
 
 # Recompile fluid-synth without CoreAudio per issues #649
 # Changes to fluid-synth.rb must be pushed to URL prior to use
 url=$(git remote get-url origin)
 branch=$(git symbolic-ref --short HEAD)
-brew install --build-from-source $url/raw/$branch/cmake/apple/fluid-synth.rb
+brew install --build-from-source "$url"/raw/"$branch"/cmake/apple/fluid-synth.rb
```

Comment:
The tool has nothing against `"$url/raw/$branch/cmake/apple/fluid-synth.rb"` ! I guess it would work too.

It has a problem with: `$url/raw/$branch/cmake/apple/fluid-synth.rb` saying: `-- SC2086: Double quote to prevent globbing and word splitting.`

See [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086).

I treated error/warning like this using search/replace, so, indeed, when there are multiple occurrence on the same line I could have use only one pair of `"`. With my coloration it's not hard to read at all, but yes, here on GitHub it looks bad.

Wiil fix this.